roles/openshift_hosted/templates/registry_config.j2

```diff
@@ -66,13 +66,13 @@ middleware:
   - name: openshift
     options:
       pullthrough: {{ openshift.hosted.registry.pullthrough | default(true) }}
-{% if openshift.hosted.registry.storage.provider == 's3' and 'cloudfront' in openshift.hosted.registry.storage.s3 %}
+{% if openshift.hosted.registry.storage.provider == 's3' and openshift_hosted_registry_storage_s3_cloudfront is defined %}
```

Comment:
We should probably add a comment that *when* re-configuration is supported that we'll need to introspect these values from the running config if the variables are not defined.